proto/wg/cosmo/node/v1/node.proto

```diff
@@ -195,10 +195,18 @@ message MTLSConfiguration {
   bool insecureSkipVerify = 3;
 }
 
+enum GraphQLSubscriptionProtocol {
```

Comment:
Configuring the router through this is legacy because the router should provide the config (ENV, config.yaml) but on the other hand, I see great potential in it too e.g. we can control certain values from control-plane. Overwriting them in the router is always possible. I'm open for both but we should have a plan. Let's discuss.

Reply:
Another option I explored was allowing configuration options to be passed in the router URL. e.g. http://my-subgraph.example.com/graphql#subscribe_with=sse - It has the advantage that wherever we accept a subgraph URL we automatically accept any options too, but it's less discoverable than a separate field in the configuration. Keep in mind that in any case we can always override all this by generating a static router configuration locally on demand. What do you think about the URL approach?

Reply:
I like it but I'd not ship it in the initial version. My question here is: Should we stop storing router configurations in the config and only use it for data-source information?

Reply:
As clarified. We go with the current proposal that the control plane has the power to change it.